docs/app/Examples/collections/Form/Validation/FormValidationExamples.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection';
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample';
+import {Message} from 'stardust';
+
+export default class FormValidationExamples extends Component {
+  render() {
+    return (
+      <ExampleSection title='Usage'>
+        <ComponentExample
+          title='Specifying Validation Rules'
+          description='Pass in a validation object with the rules required to validate your form.'
+          examplePath='collections/Form/Validation/FormSpecifyingValidationRulesExample'
+        />
+        <ComponentExample
+          title='Validating on Blur and other Events'
+          description={`
```

Comment:
Oh yes, super useful in child process scripts:

```js
import cp from 'child_process'
import path from 'path'

const dir = path.resolve(__dirname, '../some/place')
cp.exec(`
  mkdir -p ${dir}
  cd ${dir}
  echo "yo, we created and entered ${dir}.
`)
```

The line breaks are preserved characters so the commands are entered and submitted exactly as they look.  Plus, vars in your scripts nice!